community-operators/node-problem-detector/node-problem-detector.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,171 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    categories: openshift optional
+    certifiedLevel: Primed
+    containerImage: registry.svc.ci.openshift.org/openshift/origin-v4.0:node-problem-detector-operator
+    createdAt: 2019/01/31
+    description: An operator to run the OpenShift Node Problem Detector
+    healthIndex: B
+    repository: https://github.com/openshift/node-problem-detector-operator
+    support: Red Hat
+  name: node-problem-detector.v0.0.1
+  namespace: openshift-node-problem-detector
```

Comment:
nit: This should be set to `placeholder`

Reply:
This operator is a singleton and we plan to support only installs into the `openshift-node-problem-detector` namespace. Is it okay to leave that or is there a reason it needs to be `placeholder`?

Reply:
When installing via Operator Hub, the namespace field will be overwritten with the install namespace as defined by the selected Operator Group. If the operator must be installed to the `openshift-node-problem-detector` namespace then the user would create the namespace and via the UI will be able to add the operator to that namespace. Will the operator fail to install in `openshift-some-other-namespace`, for example?

Reply:
It shouldn't fail, but we don't test or support installing to another ns.

Reply:
Could you add those instructions into the description of the CSV? That way users can view it in the Console UI